task-sdk/src/airflow/sdk/execution_time/secrets_masker.py

```diff
@@ -66,6 +66,11 @@
 """Names of fields (Connection extra, Variable key name etc.) that are deemed sensitive"""
 
 SECRETS_TO_SKIP_MASKING_FOR_TESTS = {"airflow"}
+# Common terms that should be excluded from masking in production
+SECRETS_TO_SKIP_MASKING = {"airflow"}
```

Comment:
If we are never masking Airflow we don't need both these sets

Reply:
Makes complete sense since both sets only contain 'Airflow.' I'll keep just one for now.